contracts/contracts/ValidatorSet.sol

```diff
@@ -67,6 +69,14 @@ contract ValidatorSet {
         return true;
     }
 
+    function getLengthOfEpochStartHeights() external view returns(uint) {
+        return epochStartHeights.length;
+    }
+
+    function getLengthOfEpochStartToValidatorsMapping(uint _blockNumber) external view returns(uint) {
```

Comment:
Is this the number of validators of a given epoch?
If so, shall it be called `numberOfValidatorsInEpoch`

Reply:
See the [conversation earlier](https://github.com/trustlines-protocol/blockchain/pull/74#discussion_r282804941).